public/app/plugins/datasource/loki/tracking.ts

```diff
@@ -182,6 +183,27 @@ export function trackQuery(
       time_range_to: request?.range?.to?.toISOString(),
       time_taken: Date.now() - startTime.getTime(),
       bytes_processed: totalBytes,
+      ...extraPayload,
+    });
+  }
+}
+
+export function trackGroupedQueries(
+  response: DataQueryResponse,
+  requests: LokiGroupedRequest[],
+  startTime: Date
+): void {
+  const splittingPayload = {
+    splitting_groups: requests.length,
```

Comment:
I would add  something like `is_split` boolean to make it easier when querying data to filter out all split queries.

Reply:
Good idea!

Reply:
Added.